svm-transaction/src/nonce_extraction.rs

```diff
@@ -57,6 +51,15 @@ pub fn get_ix_signers(message: &impl SVMMessage, index: usize) -> impl Iterator<
         })
 }
 
+/// Serialized value of [`SystemInstruction::AdvanceNonceAccount`].
+const SERIALIZED_ADVANCE_NONCE_ACCOUNT: [u8; 4] = 4u32.to_le_bytes();
```

Comment:
Using constant here avoids bincode dependency.
The correctness is checked in the tests.